product/views/ManageIQ_Providers_InfraManager_Template.yaml

```diff
@@ -27,7 +27,6 @@ cols:
 - last_scan_on
 - region_description
 - product_name
-- os_image_name
```

Comment:
os_image_name being last is a pattern so that we can show the os icons in the table.  They should not affect sort, and they are definitely "valid"

Reply:
added back